### DISCUSSION TITLE
Visual Studio version for Copilot Chat?

### DISCUSSION THREAD

**Discussion by masters3d** [community/community#51412] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I have access to Copilot Chat using VS Code Insider and Copilot Nightly. 

What version of Visual Studio and Copilot do I need for visual studio? Or is this still not release yet?

**Comment by Akash1134** [comment]
Hi @masters3d Currently, Copilot Chat is available as a public preview and you can sign up for the waitlist of Technical Preview or Beta release [here](https://github.com/github-copilot/chat_waitlist_signup). For Copilot X, please make sure that you are using the latest [VS Code Insiders](https://code.visualstudio.com/insiders/) and the latest [GitHub Copilot Nightly extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly)

**Reply by masters3d** [reply]
Yes. What about Visual Studio? (not VS Code)

**Reply by ayim** [reply]
@masters3d you can join the waitlist for GitHub Copilot chat for Visual Studio 2022 here:
https://devblogs.microsoft.com/visualstudio/github-copilot-chat-for-visual-studio-2022/

**Reply by lokitoth** [reply]
@ayim - When I chase that sign up for preview link, it just takes me to the "you already have access" page on GH. Is there something else I need to set up to enable Chat in Big Visual Studio?

**Reply by warnov** [reply]
The same happens to me, lokitoh. Finding out how to enable copilot chat in VS2022 after being accepted in the beta is tough.

**Reply by ReynardoEW** [reply, marked as answer]
https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.VSGitHubCopilot
